### PR TITLE
fix: unorphaned `avm/res/api-management/service`

### PR DESCRIPTION
**Pull request by eriqua**
## Description

Unorphaned `avm/res/api-management/service`

### CODE REVIEW

File: avm/res/api-management/service/README.md [hunk 1]
```diff
@@ -1,10 +1,5 @@
 # API Management Services `[Microsoft.ApiManagement/service]`
 
-> ⚠️THIS MODULE IS CURRENTLY ORPHANED.⚠️
```
Comment (eriqua) [inline avm/res/api-management/service/README.md:3]: Waiting for @tony-box membership to relevant avm groups

**Review by AlexanderSehr — APPROVED** [review]

File: avm/res/api-management/service/README.md (hunk 1 above)
Comment (krbar) [reply avm/res/api-management/service/README.md:3]: @eriqua The membership is updated

Comment (eriqua) [reply avm/res/api-management/service/README.md:3]: Super, I'll ask @tony-box to kindly approve :)

**Review by tony-box — APPROVED** [review]
Looks good, thanks!

**Review by tony-box — APPROVED** [review]

**Pull request merged**